### DISCUSSION TITLE
GNOME Shell

### DISCUSSION THREAD

**Discussion by Dreamsorcerer** [brndnmtthws/conky#1847] [General]
> @Dreamsorcerer I can't find a single issue where they said anything other than:
> 
>     [No there is not, we don't intend to support third party panels, lock screens, notification UI's etc.](https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/1141)

@Caellian Just to respond to that, the 'WON'T FIX' link in your wiki goes to https://gitlab.gnome.org/GNOME/mutter/-/issues/973

> What can be done, however, is to make it possible for libmutter users, including GNOME Shell, to implement their own Wayland protocols. That way some compositor that wants to enable the user to plug in whatever panel, notification popup provider, etc it wants, can implement whatever protocol would be necessary for doing so.

Following the issue created at the end of that one takes you to https://gitlab.gnome.org/GNOME/mutter/-/issues/1053, which was closed as complete last week.

**Comment by Caellian** [comment]
Thanks, updated the wiki.
